app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -41,9 +41,10 @@ const MakeDecision = ({ data }) => {
   const { formatMessage } = useIntl();
   const dispatch = useDispatch();
   const activeProject = useSelector(activeProjectSelector);
-  const itemData = data.item;
+  const itemsData = data.items && data.items.length === 1 ? data.items[0] : data.items;
```

Comment:
This can be simplified by providing just an array with items in different cases (pass `items: [item]` for single operation or `items: items` for bulk operation).

Reply:
I further use the logic of checking 'itemData' if it is an array.